go/consensus/tendermint/full/full.go

```diff
@@ -570,6 +570,7 @@ func (t *fullService) lazyInit() error {
 	tenderConfig.Consensus.CreateEmptyBlocks = true
 	tenderConfig.Consensus.CreateEmptyBlocksInterval = emptyBlockInterval
 	tenderConfig.Consensus.DebugUnsafeReplayRecoverCorruptedWAL = viper.GetBool(CfgDebugUnsafeReplayRecoverCorruptedWAL) && cmflags.DebugDontBlameOasis()
+	tenderConfig.Mempool.Version = tmconfig.MempoolV1
```

Comment:
What is the default?

Reply:
The default is V0: https://github.com/oasisprotocol/tendermint/blob/v0.34.21-oasis/config/config.go#L738